frontend/src/state/reducers/schedulesReducer.ts

```diff
@@ -0,0 +1,67 @@
+import { NamedSchedule } from "../../models/types";
+import { mockData } from "../../data/mockData";
+import produce from "immer";
+import { getType } from "typesafe-actions";
+import { SchedulesAction } from "../actions";
+import {
+  addScheduleAction,
+  removeScheduleAction,
+  setActiveScheduleAction,
+} from "../actions/schedulesActions";
+
+export interface SchedulesState {
+  activeSchedule: number;
+  schedules: NamedSchedule[];
+}
+
```

Comment:
I think schedulesReducer clashes a little too closely with scheduleReducer. Could you change it to something like plansReducer?